app/views/publishers/expired_auth_token.html.slim

```diff
@@ -0,0 +1,19 @@
+- content_for(:navbar_content) do
+
+.publisher-domain-panel.publisher-panel.col-center
+  .publisher-domain-name
+    | &nbsp;
+
+.expired-token-panel.col-center
+  .sub-panel.col-center
+    .row.col-center
+      .col.col-md-12
+        h3.text-center= t("publishers.login_link_expired_title")
+    .row
+      .col.col-md-8.col-md-offset-2.text-center.col-center
+        p= t("publishers.login_link_expired_message")
+        = form_for @publisher, {method: :post, url: create_auth_token_publishers_path} do |f|
+          = f.hidden_field(:brave_publisher_id, :value => @publisher.brave_publisher_id)
+          = f.hidden_field(:email, :value => @publisher.email)
```

Comment:
minor: no captcha tags, so in a very edge case users would have to fill out the captcha form. that's okay i think